features/login.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Login behavior
+@QA
+Scenario: Ensure login shows proper error messages
+    Then I install the ccz app at "test_list_search.ccz"
+
+    # normal login
+    Then I login with username "user_with_no_data" and password "123"
+    # TODO: consider counting the home screen buttons
+    Then I logout
+
+    # login in landscape mode
+    # which is way more annoying than landscape mode...
+    Then I rotate to landscape
+    Then I close the keyboard
+    Then I clear input field number 1
+    Then I close the keyboard
+    Then I clear input field number 2
+    Then I close the keyboard
+    Then I enter "user_with_no_data" into input field number 1
+    Then I close the keyboard
+    Then I enter "123" into input field number 2
+    Then I close the keyboard
+    Then I press login
+    Then I wait for progress
```

Comment:
@phillipm The test is failing for me at this line with ```map android.widget.ProgressBar, query failed because: Could not detect visibility of sh.calaba.instrumentationbackend.query.ui.UIObjectView@5da4cbe
       (RuntimeError)```